pydantic/read_env_file.py

```diff
@@ -0,0 +1,23 @@
+import re
+from pathlib import Path
+from typing import Dict
+
+
+class EnvFileError(ValueError):
+    pass
+
+
+def read_env_file(file_path: Path, case_sensitive: bool = False) -> Dict[str, str]:
+    file_vars: Dict[str, str] = {}
+    for n, line in enumerate(re.split('[\r\n]+', file_path.read_text()), start=1):
+        line = line.strip()
+        if line and not line.startswith('#'):
+            if re.match(r'\s*(\w+)\s*=\s*(.+?)\s*', line):
+                key, value = line.split('=', 1)
+                key = key.strip()
```

Comment:
no need for this since it's done by the regex.

Reply:
No need for line 14? I'm doing that because a failure to match throws an error, but I don't want to throw an error for comments / blanklines.

Reply:
no need for line 17 which is where this comment is...

Reply:
Interesting, on my end the highlight is showing it as lines 14-17.